pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -119,6 +123,8 @@ func (m *Monitor) osdStatus() error {
 
 func (m *Monitor) handleOSDMarkedOut(outOSDid int) error {
 	label := fmt.Sprintf("ceph-osd-id=%d", outOSDid)
+	opconfig.ConditionExport(m.context, m.clusterName, m.crdName,
```

Comment:
Let's not worry about the condition for shrinking the cluster. This is really not a common scenario. We only remove the OSD when its data has already migrated to other OSDs, so there is really nothing for the user to worry about at this point.

Reply:
Okay. So for now just stick with Expanding only right?

Reply:
@travisn ^

Reply:
Per my other comment, I don't think expanding or shrinking conditions are needed.